src/components/NewLocationPage/ChartTabs/ChartTab.style.tsx

```diff
@@ -40,6 +40,7 @@ export const Tab = styled(MuiTab).attrs(props => ({
 }))`
   line-height: 1;
   text-align: left;
+  max-width: 372px;
```

Comment:
This modifies the max-width of the title/header data above the chart--without this, it was cutoff and squished. This doesn't feel great to edit, though, so if there's another way to fix this we should use that.

I arbitrarily chose 372 because it's divisible by 12, but idk if we're using any sort of grid system where that matters. 

before:
<img width="609" alt="image" src="https://user-images.githubusercontent.com/55333380/150407018-f2638be2-d1c5-4b74-85c2-96e076e6de18.png">

after:
<img width="609" alt="image" src="https://user-images.githubusercontent.com/55333380/150407152-bf0b5c23-7264-43a1-8cb2-7c8bbb6057bb.png">

Reply:
Hrm. I'm not sure I understand this change.
1. If the goal was to make the tab wider, I'd expect you to maybe add a min-width rather than a max-width.
2. When I look in Chrome dev tools it looks like the style isn't even being applied because it's overridden by a `width` specified in a more specific rule:

![image](https://user-images.githubusercontent.com/206364/150431994-97d5e735-94bf-4a6d-bb60-4c66edf5185e.png)

Also, depending on the screen size I can still make it look kinda' funky, e.g. according to chrome dev tools mobile mode, on an iPhone SE it will look like this:
![image](https://user-images.githubusercontent.com/206364/150432197-7703c8fd-52a0-423e-be2e-436992c35bfb.png)

In general I'm terrible at CSS but it feels like we need some margin or padding maybe?  Not sure. Maybe ask Pablo if you get stuck.

Reply:
hmm yeah, good flag--looking back I'm not sure how/why this works (or how to fix it on mobile). I've experimented some, and it looks like using `min-width` instead of `max-width` also works, but this doesn't really address the problem. 

I'll probably ask at stand-up today if anyone could lend a hand

Reply:
as for what the change did, I think it expands the div that contains the three vaccination % divs, which seems to allow them to spread out more

<img width="1092" alt="image" src="https://user-images.githubusercontent.com/55333380/150810901-4bcad8e1-1bc8-4982-8172-2b30c92a7497.png">

<img width="1092" alt="image" src="https://user-images.githubusercontent.com/55333380/150811166-fce49980-4d43-40d6-8064-57bca0921b38.png">

Reply:
I can take this UI change, it's a bit more complex than it looks